dnn/data/converters/get_purdue.py

```diff
@@ -0,0 +1,97 @@
+import json
+import xml.dom.minidom as minidom
+
+import requests
+from absl import app, flags
+from absl.flags import FLAGS
+from tqdm import tqdm
+import os
+import time
+
+flags.DEFINE_string('input', '../Purdue Dataset/label.json',
+                    'the path for input ROCO dataset (please unzip by yourself)')
+flags.DEFINE_string('output', '../Purdue Dataset',
+                    'the path for input ROCO dataset (please unzip by yourself)')
```

Comment:
This default path `../<xxx>` is not very intuitive. If a user run the script and look into the directory, there will be nothing. Usually there are two choices:

1. Remove a default path, and make it a required command line argument
2. Make the path relative to current directory, ie `./<xxx>`

I strongly prefer the first one.

Reply:
Do you mean put datasets in `iRM_Autonomy_2020\dnn\data\converters` ?

Reply:
`absl` requires a default value for every flag, so the first choice also does not exist.